client/Bootstrap/README.md

```diff
@@ -6,7 +6,7 @@ While it's own directory, the code in this directory should be tested and develo
 
 ## Test approach
 
-To be defined in https://github.com/strimzi/strimzi-ui/issues/7
+Bootstrap should be tested in a behavioural manor. See [Behavioural Driven Development](../../docs/Test.md#style-of-test).
```

Comment:
I think it would be worth adding here that the tests for `Bootstrap` level components (and the whole UI more generally) are kept in the `e2e` folder (with a link to it).

Reply:
Happy to say bootstrap needs to only be tested through E2E as that is much more meaningful than trying to make fiddly unit tests. Hesitant about the navigation pieces being E2E only but willing to at least try, depending on its implementation. @dlabaj do you have any thoughts on this?